io.github.fastrizwaan.WebkitWord.yaml

```diff
@@ -0,0 +1,24 @@
+id: io.github.fastrizwaan.WebkitWord
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '48'
+command: webkitword
+base: org.libreoffice.LibreOffice
```

Comment:
why does it use libreoffice as base?

Reply:
what changed here? This wasn't present in the initial submission.

Reply:
uses libreoffice to convert docx, odt, rtf files to html. and usess it to export to save them.

Reply:
libreoffice was there right from the 1st submisison.

Reply:
I see, looks like I misremembered sorry.